README.md

```diff
@@ -182,6 +192,12 @@ After running the generation the output would be
                                 generated/
                                     maven/
                                         All generated .class files for the maven-4.0.0.xsd schema in here
+                        sixthpackage/
```

Comment:
I don't think the output locations here (`$buildDir/generated-sources/main/xjc/com/sixthpackage/generated/part1` and `$buildDir/generated-sources/main/xjc/com/sixthpackage/generated/part2`) are accurate based on the configuration

```groovy
someSixthSchema {
  schemaDir = 'sixth-schemas'
  schemaRootDir = 'misc/resources/schemas'
  onePassMode = true
}
```

1. The `someSixthSchema` configuration doesn't define the `javaPackageName = 'com.sixthpackage.generated'`, which is where you are saying the output should be generated to
1. According to your configuration, all of the schemas in `misc/resources/schemas/sixth-schemas` would be generated with the same package
    - i.e. All generated classes from both `somesixthschema1.xsd` and `somesixthschema2.xsd` would be generated in the same package in the same output directory
    - The change in output directories for each schema _could_ be controlled by a binding file, but no binding file was specified in this particular configuration.

Reply:
In fact `xjc` can generate separate packages even if no bindings file is specified. It uses the target namespace defined in the schema file to locate a Java package. This is the case in the integration test I've added. Packages `com.github.edeandrea.xjcplugin.generated.schemadirwith2relatedschemas.type`and `com.github.edeandrea.xjcplugin.generated.schemadirwith2relatedschemas.product` are used and are only defined by the schema files.
If the `javaPackageName` is defined, then all the classes will be generated in this package.

Reply:
You know what you are correct. Sorry I did not look into the schema files.